applications/FluidDynamicsApplication/python_scripts/fluid_dynamics_analysis.py

```diff
@@ -71,7 +71,8 @@ def _GetOrderOfProcessesInitialization(self):
         return ["gravity",
                 "initial_conditions_process_list",
                 "boundary_conditions_process_list",
-                "auxiliar_process_list"]
+                "auxiliar_process_list",
+                "mesh_adaptivity_processes"]
```

Comment:
I would really really not add anything unless absolutely necessary
I think the order of when those processes are done doesn't matter

Reply:
I think is adequate to have a category just for mesh adaptivity

Reply:
read the doc in the baseclass:
~~~
This function can be overridden in derived classes if the order of initialization for the processes matters
~~~

this should NOT be necessary!!!

The names of the blocks don't matter otherwise

Reply:
The problem is then the order?, in any case remeshing should be the last thing to do